packages/opentelemetry-exporter-trace-otlp-grpc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/exporter-trace-otlp-grpc",
-  "version": "0.27.0",
+  "version": "1.0.0",
```

Comment:
Version must match the lerna.json version

Reply:
Thanks. Changed that. I did think that we version the packages independently though... :thinking:

Reply:
No we only have 2 versions, the experimental and the stable versions are different. You may be thinking of contrib where the versions are completely independent.